float32 resolution
---
sensor_msgs/PointCloud2 cloud
